uber/sep_commands.py

```diff
@@ -14,19 +14,10 @@ def print_config():
 @entry_point
 def insert_admin():
     with Session() as session:
```

Comment:
Since you're no longer doing database stuff in this method, you no longer need the ``with Session() as session`` block.  Doesn't Doesn't hurt to leave it in, it's just unnecessary.